server/views/pages/reports/history/type.njk

```diff
@@ -41,7 +41,7 @@
 
                   {% if inset %}
                     <div class="app-response--inset">
-                      {{ response.response }}
+                      {{ response.label or response.response }}
```

Comment:
aren't the labels always there? 🤔

Reply:
yeah you’re right. db has no `''` nor `null` so i have redone it 0d203e8c72c79e7ec0799efb1eb28ed4e2f91370